Add text changes for Log Search in Timeboards (#5040)
DataDog/documentation: content/en/graphing/dashboards/timeboard.md

2. Click on the pencil icon to edit the title and description
3. Click the checkmark to save changes

## Event Correlation 

Event Correlation refers to overlaying events on top of a dashboard graph. You can set up correlation at two different times: when you set up the dashboard, or ad-hoc when you view the dashboard.

### On an individual graph (At design time)

{{< img src="graphing/dashboards/guides-eventcorrelation-screenboard.png" alt="guides-eventcorrelation-screenboard" responsive="true" style="width:90%;">}}

Set up event correlation at design time by editing any graph on both Timeboards and Screenboards and adding events to the graph. You can find details about adding events [using the UI][1] or via the JSON interface further down the page.

### On whole dashboard

{{< img src="graphing/dashboards/open-search-event.png" alt="guides event correlation" responsive="true" style="width:75%;">}}

> Set up event correlation at view time by clicking on the "Search Events of Logs" link in the upper left hand side of the dashboard, and select "Events". Next, type a query in the search box at the top left of any Timeboard dashboard window. This replaces any events added at design time, but applies the events to all graphs on that particular dashboard as an overlay. 

Reviewer: ```suggestion
Set up event correlation at view time by clicking on the *Search Events of Logs* link in the upper left hand side of the dashboard, and select *Events*. Next, type a query in the search box at the top left of any Timeboard dashboard window. This replaces any events added at design time, but applies the events to all graphs on that particular dashboard as an overlay. 
```

## Correlation between Logs and Metrics

### Search for Logs in Timeboards

> Overlaying logs in your dashboard is easy. Click on the "Search Events of Logs" link in the upper left hand side of the dashboard, and select "Logs". This overlays the frequency of logs on your timeseries graphs. Click on the individual log line to view the full content.

Reviewer: ```suggestion
Click on the *Search Events of Logs* link in the upper left hand side of the Timeboard, and select **Logs**. This overlays the frequency of logs on your timeseries widgets. Click on an individual log line to view its full content.
```
